content/codes.md

```diff
@@ -848,6 +841,13 @@ toc: true
 - [RfSurfHmc](https://github.com/nqdu/RfSurfHmc): Joint inversion of Receiver
   Function and Surface Wave Disperion by Hamilton Monte Carlo Method
 
+### Waveform Inversion
```

Comment:
I am still not satisfied with this title, which might be confused with full waveform inversion.